### PR TITLE
Update documentation about 2 settings screens

### PR DESCRIPTION
**Pull request by imath**
- Adds a documentation page to explain the BP Components Administration screen.
- Adds a documentation page to explain the BP Options Administration screen. 

Trac ticket: https://buddypress.trac.wordpress.org/ticket/9004

---
**This Pull Request is for code review only. Please keep all other discussion in the BuddyPress Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the WordPress Core Handbook for more details.**

### CODE REVIEW

File: docs/user/administration/settings/components.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# BuddyPress Components settings
+
+The BuddyPress Core & Members components are required ones and are always enabled. The first time you activate ByddyPress, 4 optional components are activated for you, by default:
```
Comment (krupal-panchal) [inline docs/user/administration/settings/components.md:3]: BuddyPress is misspelled here.

Comment (imath) [reply docs/user/administration/settings/components.md:3]: Hi, thanks for your check, I will update the PR asap 👌

Comment (imath) [inline docs/user/administration/settings/components.md:3]:
```suggestion
The BuddyPress Core & Members components are required ones and are always enabled. The first time you activate BuddyPress, 4 optional components are activated for you, by default: 
```

**Review by imath — COMMENTED** [review]
Prepare @krupal-panchal fix

**Review by krupal-panchal — APPROVED** [review]
LGTM

**Pull request closed without merging**